util/pyclient/test_msgs.py

```diff
@@ -14,6 +14,119 @@
 import struct
 
 import bft_msgs
+import replica_specific_info as rsi
+
+
+class TestSpecificReplyInfoManager(unittest.TestCase):
```

Comment:
Great tests!

The name can probably be simplified to `TestRepliesManager` though.

Reply:
done